<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="MANIGUJJAR" failures="0" tests="2" name="SIgnupPage.TestNGSignUP" time="1.072" errors="1" timestamp="2020-12-15T03:05:11 PKT" skipped="0">
  <testcase classname="SIgnupPage.TestNGSignUP" name="TestNg" time="0.007"/>
  <system-out/>
  <testcase classname="SIgnupPage.TestNGSignUP" name="test4data" time="1.065">
    <error message="
[public void SIgnupPage.TestNGSignUP.test4data() throws java.lang.Exception] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: test4data([])
Arguments: [(java.lang.String) d32@gmail.com,(java.lang.String) AB,(java.lang.String) AB,(java.lang.String) 12345,(java.lang.String) 11,(java.lang.String) April,(java.lang.String) 1997,(java.lang.String) ABC,(java.lang.String) Florida,(java.lang.String) Florida,(java.lang.String) 55270,(java.lang.String) United States,(java.lang.String) 3331234567]" type="org.testng.internal.reflect.MethodMatcherException">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
[public void SIgnupPage.TestNGSignUP.test4data() throws java.lang.Exception] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: test4data([])
Arguments: [(java.lang.String) d32@gmail.com,(java.lang.String) AB,(java.lang.String) AB,(java.lang.String) 12345,(java.lang.String) 11,(java.lang.String) April,(java.lang.String) 1997,(java.lang.String) ABC,(java.lang.String) Florida,(java.lang.String) Florida,(java.lang.String) 55270,(java.lang.String) United States,(java.lang.String) 3331234567]
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:43)
at org.testng.internal.Parameters.injectParameters(Parameters.java:905)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:34)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- test4data -->
  <system-out/>
</testsuite> <!-- SIgnupPage.TestNGSignUP -->
